internal/cli/delete/delete.go

```diff
@@ -151,6 +141,22 @@ func (o *DeleteOptions) complete() error {
 	if err != nil {
 		return err
 	}
+	// confirm names to delete, use ConfirmedNames first, if it is empty, use Names
+	if !o.AutoApprove {
```

Comment:
attribute `o.ConfirmedNames` is never used, is it expected?

Reply:
> attribute `o.ConfirmedNames` is never used, is it expected?

fix it! Add confirmation for the names selected by the label selector in the previous logic.